posthog/api/query.py

```diff
@@ -183,3 +192,121 @@ def _tag_client_query_id(self, query_id: str | None):
 
         tag_queries(client_query_id=query_id)
         set_tag("client_query_id", query_id)
+
+
+ASYNC_FALLBACK_TO_POLLING_TIMEOUT = 50
+
+
+async def query_async(request: Request, *args, **kwargs) -> HttpResponse:
```

Comment:
Let's not overload the term `async` so much, because async queries already are a thing in the existing query endpoint. This is rather an UX optimization _to_ async querying.

Reply:
Yeah fair.. I'm just going to roll it out, then I'll make this the default way for queries to happen once we've rolled it out